ts/Core/Series/DataLabel.ts

```diff
@@ -458,7 +458,9 @@ namespace DataLabel {
         const series = this,
             chart = series.chart,
             seriesOptions = series.options,
-            hasRendered = series.hasRendered || 0,
+            hasRendered = series.type === 'networkgraph' ?
```

Comment:
I don't like this exception here, but I did not yet come up with a better solution (other than overriding the entire `drawDataLabels` function in `NetworkgraphSeries.ts`.

Reply:
We can't leave it like this. We already override `drawDataLabels` in `NetworgraphSeries.ts` so it's not a problem

Reply:
Hint:
You can set temporarily `hasRendered` and then restore it:
```ts
drawDL() {
  const hasRendered = this.hasRendered;

  this.hasRendered = true;
  super();
  this.hasRendered = hasRendered;
}

```

Reply:
You're right! Thanks!